### PR TITLE
En 8398 tests transactions

### PR DESCRIPTION
**Pull request by miiu96**
Added a lot of integration tests that verify if the transaction processor works as exepected.

### CODE REVIEW

File: integrationTests/vm/testInitializer.go [hunk 1]
```diff
@@ -5,11 +5,14 @@ package vm
 import (
 	"encoding/hex"
 	"fmt"
+	"github.com/ElrondNetwork/elrond-go/process/block/postprocess"
 	"math"
 	"math/big"
 	"strconv"
 	"testing"
 
+	"github.com/ElrondNetwork/elrond-go/process/block/preprocess"
```
Comment (sasurobert) [inline integrationTests/vm/testInitializer.go:14]: goimports

**Review by sasurobert — CHANGES_REQUESTED** [review]

File: process/transaction/export_test.go [hunk 2]
```diff
@@ -57,9 +58,10 @@ func (txProc *txProcessor) ProcessUserTx(
 
 func (txProc *txProcessor) TakeMoveBalanceCostOutOfUser(
```
Comment (SebastianMarian) [inline process/transaction/export_test.go:59]: Rename also this method ProcessMoveBalanceCostRelayedUserTx

File: integrationTests/vm/txsFee/builtInFunctions_test.go [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+	_, err = testContext.Accounts.Commit()
+	require.Nil(t, err)
+
+	checkOwnerAddr(t, &testContext, scAddress, newOwner)
+
+	expectedBalance := big.NewInt(88180)
+	vm.TestAccount(t, testContext.Accounts, owner, 2, expectedBalance)
+
+	// check accumulated fees
+	accumulatedFee := testContext.TxFeeHandler.GetAccumulatedFees()
```
Comment (SebastianMarian) [inline integrationTests/vm/txsFee/builtInFunctions_test.go:43]: You can search everywhere after this message: // check accumulated fees , and replace the below variable name from 
accumulatedFee to accumulatedFees

File: integrationTests/vm/testInitializer.go (hunk 1 above)
Comment (miiu96) [reply integrationTests/vm/testInitializer.go:14]: done.

File: process/transaction/export_test.go (hunk 2 above)
Comment (miiu96) [reply process/transaction/export_test.go:59]: done.

File: integrationTests/vm/txsFee/builtInFunctions_test.go (hunk 3 above)
Comment (miiu96) [reply integrationTests/vm/txsFee/builtInFunctions_test.go:41]: done.

**Review by sasurobert — APPROVED** [review]

File: integrationTests/vm/txsFee/moveBalance_test.go [hunk 4]
```diff
@@ -134,12 +134,12 @@ func TestMoveBalanceHigherNonceShouldNotConsumeGas(t *testing.T) {
 	vm.TestAccount(t, testContext.Accounts, sndAddr, 0, expectedBalance)
 
 	// check accumulated fees
-	accumulatedFee := testContext.TxFeeHandler.GetAccumulatedFees()
-	require.Equal(t, big.NewInt(0), accumulatedFee)
+	accumulatedFeess := testContext.TxFeeHandler.GetAccumulatedFees()
```
Comment (SebastianMarian) [inline integrationTests/vm/txsFee/moveBalance_test.go:137]: *feess

Comment (miiu96) [reply integrationTests/vm/txsFee/moveBalance_test.go:137]: done.

**Review by SebastianMarian — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Review by LucianMincu — APPROVED** [review]
👍

**Pull request merged**